nohup: ignoring input
[I 09:30:26.957 NotebookApp] Serving notebooks from local directory: /data/1xiu/project/pythonPro
[I 09:30:26.958 NotebookApp] 0 active kernels 
[I 09:30:26.958 NotebookApp] The Jupyter Notebook is running at: https://[all ip addresses on your system]:8888/
[I 09:30:26.958 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[I 09:30:38.915 NotebookApp] Kernel started: 3c08892f-3a2c-4865-8227-cae5fe1ebb91
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.version
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.version
Setting default log level to "WARN".
To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[I 09:30:54.353 NotebookApp] Kernel started: 5a2863c5-4c0d-4219-981b-372dd00f191c
Setting default log level to "WARN".
To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
17/07/19 09:31:01 WARN AbstractLifeCycle: FAILED ServerConnector@6b3d1257{HTTP/1.1}{0.0.0.0:4040}: java.net.BindException: Address already in use
java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:223)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:74)
	at org.spark_project.jetty.server.ServerConnector.open(ServerConnector.java:321)
	at org.spark_project.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80)
	at org.spark_project.jetty.server.ServerConnector.doStart(ServerConnector.java:236)
	at org.spark_project.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.spark_project.jetty.server.Server.doStart(Server.java:366)
	at org.spark_project.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.apache.spark.ui.JettyUtils$.org$apache$spark$ui$JettyUtils$$connect$1(JettyUtils.scala:349)
	at org.apache.spark.ui.JettyUtils$$anonfun$5.apply(JettyUtils.scala:359)
	at org.apache.spark.ui.JettyUtils$$anonfun$5.apply(JettyUtils.scala:359)
	at org.apache.spark.util.Utils$$anonfun$startServiceOnPort$1.apply$mcVI$sp(Utils.scala:2195)
	at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:160)
	at org.apache.spark.util.Utils$.startServiceOnPort(Utils.scala:2186)
	at org.apache.spark.ui.JettyUtils$.startJettyServer(JettyUtils.scala:359)
	at org.apache.spark.ui.WebUI.bind(WebUI.scala:140)
	at org.apache.spark.SparkContext$$anonfun$10.apply(SparkContext.scala:460)
	at org.apache.spark.SparkContext$$anonfun$10.apply(SparkContext.scala:460)
	at scala.Option.foreach(Option.scala:257)
	at org.apache.spark.SparkContext.<init>(SparkContext.scala:460)
	at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
	at py4j.Gateway.invoke(Gateway.java:236)
	at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
	at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
	at py4j.GatewayConnection.run(GatewayConnection.java:214)
	at java.lang.Thread.run(Thread.java:745)
17/07/19 09:31:01 WARN AbstractLifeCycle: FAILED org.spark_project.jetty.server.Server@145ad4ff: java.net.BindException: Address already in use
java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:223)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:74)
	at org.spark_project.jetty.server.ServerConnector.open(ServerConnector.java:321)
	at org.spark_project.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80)
	at org.spark_project.jetty.server.ServerConnector.doStart(ServerConnector.java:236)
	at org.spark_project.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.spark_project.jetty.server.Server.doStart(Server.java:366)
	at org.spark_project.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.apache.spark.ui.JettyUtils$.org$apache$spark$ui$JettyUtils$$connect$1(JettyUtils.scala:349)
	at org.apache.spark.ui.JettyUtils$$anonfun$5.apply(JettyUtils.scala:359)
	at org.apache.spark.ui.JettyUtils$$anonfun$5.apply(JettyUtils.scala:359)
	at org.apache.spark.util.Utils$$anonfun$startServiceOnPort$1.apply$mcVI$sp(Utils.scala:2195)
	at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:160)
	at org.apache.spark.util.Utils$.startServiceOnPort(Utils.scala:2186)
	at org.apache.spark.ui.JettyUtils$.startJettyServer(JettyUtils.scala:359)
	at org.apache.spark.ui.WebUI.bind(WebUI.scala:140)
	at org.apache.spark.SparkContext$$anonfun$10.apply(SparkContext.scala:460)
	at org.apache.spark.SparkContext$$anonfun$10.apply(SparkContext.scala:460)
	at scala.Option.foreach(Option.scala:257)
	at org.apache.spark.SparkContext.<init>(SparkContext.scala:460)
	at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
	at py4j.Gateway.invoke(Gateway.java:236)
	at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
	at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
	at py4j.GatewayConnection.run(GatewayConnection.java:214)
	at java.lang.Thread.run(Thread.java:745)
17/07/19 09:31:01 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.version
[I 09:32:36.019 NotebookApp] Kernel shutdown: 3c08892f-3a2c-4865-8227-cae5fe1ebb91
[W 09:32:36.037 NotebookApp] delete /Test PySpark.ipynb
[I 09:33:17.311 NotebookApp] Creating new notebook in 
[E 09:33:18.280 NotebookApp] Failed to run command:
    [u'/data/1xiu/anaconda3/envs/python2.7/bin/python', u'-m', u'ipykernel_launcher', u'-f', u'/run/user/0/jupyter/kernel-19b03938-4e2f-47c7-bede-02ee474051bb.json']
        PATH='/data/1xiu/anaconda2/bin:/usr/hdp/2.5.0.0-1245/spark2/bin:/usr/lib64/qt-3.3/bin:/usr/local/jdk/bin:/usr/local/jdk/jre/bin:/usr/local/jdk:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/data/tools/scala-2.12.2/bin:/root/bin'
        with kwargs:
    {'cwd': u'/data/1xiu/project/pythonPro', 'stdin': -1, 'preexec_fn': <function <lambda> at 0x7f0be48cc578>, 'stderr': None, 'stdout': None}
    
[E 09:33:18.304 NotebookApp] Unhandled error in API request
    Traceback (most recent call last):
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/notebook/base/handlers.py", line 516, in wrapper
        result = yield gen.maybe_future(method(self, *args, **kwargs))
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/gen.py", line 1055, in run
        value = future.result()
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/concurrent.py", line 238, in result
        raise_exc_info(self._exc_info)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/gen.py", line 1063, in run
        yielded = self.gen.throw(*exc_info)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/notebook/services/sessions/handlers.py", line 75, in post
        type=mtype))
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/gen.py", line 1055, in run
        value = future.result()
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/concurrent.py", line 238, in result
        raise_exc_info(self._exc_info)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/gen.py", line 1063, in run
        yielded = self.gen.throw(*exc_info)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/notebook/services/sessions/sessionmanager.py", line 79, in create_session
        kernel_id = yield self.start_kernel_for_session(session_id, path, name, type, kernel_name)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/gen.py", line 1055, in run
        value = future.result()
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/concurrent.py", line 238, in result
        raise_exc_info(self._exc_info)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/gen.py", line 1063, in run
        yielded = self.gen.throw(*exc_info)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/notebook/services/sessions/sessionmanager.py", line 92, in start_kernel_for_session
        self.kernel_manager.start_kernel(path=kernel_path, kernel_name=kernel_name)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/gen.py", line 1055, in run
        value = future.result()
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/concurrent.py", line 238, in result
        raise_exc_info(self._exc_info)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/tornado/gen.py", line 307, in wrapper
        yielded = next(result)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/notebook/services/kernels/kernelmanager.py", line 94, in start_kernel
        super(MappingKernelManager, self).start_kernel(**kwargs)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/jupyter_client/multikernelmanager.py", line 110, in start_kernel
        km.start_kernel(**kwargs)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/jupyter_client/manager.py", line 257, in start_kernel
        **kw)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/jupyter_client/manager.py", line 203, in _launch_kernel
        return launch_kernel(kernel_cmd, **kw)
      File "/data/1xiu/anaconda2/lib/python2.7/site-packages/jupyter_client/launcher.py", line 128, in launch_kernel
        proc = Popen(cmd, **kwargs)
      File "/data/1xiu/anaconda2/lib/python2.7/subprocess.py", line 390, in __init__
        errread, errwrite)
      File "/data/1xiu/anaconda2/lib/python2.7/subprocess.py", line 1024, in _execute_child
        raise child_exception
    OSError: [Errno 2] No such file or directory
[E 09:33:18.314 NotebookApp] {
      "Origin": "https://10.110.13.207:8888", 
      "Content-Length": "94", 
      "Accept-Language": "zh-CN,zh;q=0.8,en;q=0.6", 
      "Accept-Encoding": "gzip, deflate, br", 
      "X-Xsrftoken": "2|2889046b|bf041e881b9cf54239fe0aa5bc1d709f|1500273313", 
      "Host": "10.110.13.207:8888", 
      "Accept": "application/json, text/javascript, */*; q=0.01", 
      "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Safari/537.36", 
      "Connection": "keep-alive", 
      "X-Requested-With": "XMLHttpRequest", 
      "Referer": "https://10.110.13.207:8888/notebooks/Untitled1.ipynb?kernel_name=python2.7", 
      "Alexatoolbar-Alx_ns_ph": "AlexaToolbar/alx-4.0.1", 
      "Cookie": "_xsrf=2|2889046b|bf041e881b9cf54239fe0aa5bc1d709f|1500273313; username-10-110-13-207-8889=\"2|1:0|10:1500273317|27:username-10-110-13-207-8889|44:MTk3NTM0M2VjZGZhNDRlZWIwNjE2MTExYjVjMDNhMWU=|511bce3c2933083d480d0d4d870153ed76d743db0fd4043e6627dc864b6e132b\"; username-10-110-13-207-8888=\"2|1:0|10:1500363087|27:username-10-110-13-207-8888|44:YWI2Njg3NGM5MTQ3NGRkMTliZmZlMzUyNTcxODliMzg=|3b51bd33904de6e346cb3bd3951b7a824f74554d09b7ba634e451e1e4ab8ede4\"", 
      "Content-Type": "application/json"
    }
[E 09:33:18.314 NotebookApp] 500 POST /api/sessions (10.9.11.35) 61.63ms referer=https://10.110.13.207:8888/notebooks/Untitled1.ipynb?kernel_name=python2.7
[I 09:33:25.297 NotebookApp] Creating new notebook in 
[I 09:33:26.113 NotebookApp] Kernel started: 1347013d-ceb3-4041-bd51-fbf030f22f51
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.version
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.version
[I 09:33:34.043 NotebookApp] Creating new notebook in 
[I 09:33:34.633 NotebookApp] Kernel started: dc86b1e2-9401-4508-89e8-9a7df4ecbb52
Setting default log level to "WARN".
To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[I 09:34:01.737 NotebookApp] Saving file at /Untitled3.ipynb
[I 09:34:02.063 NotebookApp] Saving file at /Untitled3.ipynb
[I 09:34:02.327 NotebookApp] Saving file at /Untitled3.ipynb
[I 09:34:02.545 NotebookApp] Saving file at /Untitled3.ipynb
[I 09:35:24.438 NotebookApp] Interrupted...
[I 09:35:24.438 NotebookApp] Shutting down kernels
[I 09:35:24.742 NotebookApp] Kernel shutdown: 1347013d-ceb3-4041-bd51-fbf030f22f51
Traceback (most recent call last):
  File "/data/1xiu/anaconda2/bin/jupyter-notebook", line 6, in <module>
    sys.exit(notebook.notebookapp.main())
  File "/data/1xiu/anaconda2/lib/python2.7/site-packages/jupyter_core/application.py", line 267, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/data/1xiu/anaconda2/lib/python2.7/site-packages/traitlets/config/application.py", line 658, in launch_instance
    app.start()
  File "/data/1xiu/anaconda2/lib/python2.7/site-packages/notebook/notebookapp.py", line 1422, in start
    self.cleanup_kernels()
  File "/data/1xiu/anaconda2/lib/python2.7/site-packages/notebook/notebookapp.py", line 1309, in cleanup_kernels
    self.kernel_manager.shutdown_all()
  File "/data/1xiu/anaconda2/lib/python2.7/site-packages/jupyter_client/multikernelmanager.py", line 160, in shutdown_all
    self.finish_shutdown(kid)
  File "/data/1xiu/anaconda2/lib/python2.7/site-packages/jupyter_client/multikernelmanager.py", line 33, in wrapped
    r = method(*args, **kwargs)
  File "/data/1xiu/anaconda2/lib/python2.7/site-packages/jupyter_client/manager.py", line 280, in finish_shutdown
    time.sleep(pollinterval)
KeyboardInterrupt
